### ISSUE TITLE
Data augmentation

### ISSUE THREAD

**Issue by MrGreensushi** [MrGreensushi/MachineLearningProject#2]
Dividere il database idn:
Training set
Cross Validation set
Test set

Scrivere codice in pyhton a modo di funzione che faccia il seguente:
- ad ogni immagine del database crea n nuove immagine con le seguenti modifiche:

1. Rotazione di 45,90,-45,-90 gradi
2. Cambio luminosità o saturare/desaturare l'immagine
